Tests/TuistKitTests/Services/Plugin/PluginArchiveServiceTests.swift

```diff
@@ -108,7 +108,9 @@ final class PluginArchiveServiceTests: TuistUnitTestCase {
         // Then
         XCTAssertEqual(invokedPackagePath, path)
         XCTAssertEqual(builtProducts, ["tuist-one", "tuist-two"])
-        XCTAssertEqual(fileArchiver.invokedZipParameters?.name, "TestPlugin.tuist-plugin.zip")
+
+        _ = verify(fileArchiver).zip(name: .value("TestPlugin.tuist-plugin.zip"))
```

Comment:
@pepicrft 
This should have been:
```swift
verify(fileArchiver).zip(name: .value("TestPlugin.tuist-plugin.zip")).called(count: .atLeastOnce)
```
Without the last part there is no verification happening.